src/util/misc/matrix.ts

```diff
@@ -166,11 +166,17 @@ export const createTranslateMatrix = (x: number, y = 0): TMat2D => [
  */
 export function createRotateMatrix(
   { angle = 0 }: TRotateMatrixArgs = {},
+  origin: Partial<XY> = {}
+): TMat2D {
+  return createRotationMatrix(degreesToRadians(angle), origin);
+}
+
+export function createRotationMatrix(
```

Comment:
can we remove this double function? we talked about it in some meeting, to keep just the one with angle

Reply:
Do you mind if I change the signature to accept angle as it does or rotation for the rad value?

Reply:
I think we have the api in angle, so no api with radiants. calcRotateMatrix is good enough.